src/assets/RestApi/rest-api.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Basics of REST API with Examples
+authorName: Prajwol Amatya
+authorAvatar: https://1.gravatar.com/avatar/de64e53c0e2cb393dd0d14ffdd53058ee9c607b35e366dd392425bd1b95a034c?size=256
+authorLink: https://github.com/prajwolamatya
+createdAt: April 16, 2024
+tags: rest, api, rest api, python, flask
+banner: https://blog.jankaritech.com/src/assets/RestApi/images/rest-api-banner.png
+---
+
+Within the field of web development, APIs, or application programming interfaces, are essential for facilitating communication across various software components. Because of its simplicity, scalability, and statelessness, REST (Representational State Transfer) APIs stand out among the other types of APIs. In this blog, we will dive into basics of building a REST API.
+
+## What is a REST API?
+REST (Representational State Transfer), serves as an architectural approach for creating networked apps. It operates via a stateless client-server communication system. In this model, clients and servers exchange representations of resources identified by URLs. Access is facilitated through standard HTTP methods such as GET, POST, PUT, DELETE, and others.
```

Comment:
```suggestion
REST API serves as an architectural approach for creating networked apps. It operates via a stateless client-server communication system. In this model, clients and servers exchange representations of resources identified by URLs. Access is facilitated through standard HTTP methods such as GET, POST, PUT, DELETE, and others.
```
Since the full form of REST has already been provided above, there is no need to repeat it again.